package.json

```diff
@@ -106,11 +106,11 @@
     "gulp-spawn-mocha": "^6.0.0",
     "gulp-stylelint": "^13.0.0",
     "gulp-uglify": "^3.0.0",
-    "handlebars": "^4.7.3",
+    "handlebars": "^4.7.6",
```

Comment:
Might want to update in notes. We have `0.4.3 → 0.4.6` instead of `4.7.3 → 4.7.6`

Reply:
Updated!